sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromBeginningImpl.java

```diff
@@ -16,10 +16,12 @@ public ChangeFeedStartFromBeginningImpl() {
     public void populatePropertyBag() {
         super.populatePropertyBag();
```

Comment:
if there is any concurrency concern. I suspect the synchronization may need to get extend here as well.

any reason this needs to be outside of synchronization scope.

also elsewhere for similar cases.

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.